docs/news.rst

```diff
@@ -19,6 +19,8 @@ Psycopg 3.2 (unreleased)
   (:ticket:`340`).
 - Add :ref:`raw-query-cursors` to execute queries using placeholders in
   PostgreSQL format (`$1`, `$2`...) (:ticket:`#560`).
+- Add `psycopg.capabilities` object to :ref:`inspect the libpq capabilities
+  <capabilities>` (:ticket:`#772`).
```

Comment:
The `_` syntax is a reST feature and works only in the same file. Because of this limitation, Sphinx added the `ref` role which allows linking to references in different files in the doc.